Scripts/swiftformat-lint.sh

```
@@ -0,0 +1,7 @@
+cd "$(dirname "$0")"/../ || exit 1
+
+if [ -z "$1" ]; then
+  swiftformat . --exclude "Sources/ULID/*,Tests/ULIDTests/*,Sources/VeinCore/VeinCore.docc/*,Sources/VeinSwiftUI/VeinSwiftUI.docc/*" --lint
+else
+  swiftformat $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)